emsdk

```diff
@@ -2210,6 +2223,8 @@ def main():
         sys.argv[i] = str(find_latest_nightly_32bit_sdk())
       elif sys.argv[i] == 'sdk-nightly-latest-64bit':
         sys.argv[i] = str(find_latest_nightly_64bit_sdk())
+      elif sys.argv[i] == 'latest-upstream' or sys.argv[i] == 'latest-clang-upstream':
```

Comment:
Should we update readme.md to let people know this exists, or wait until that's less-beta? Should probably be done in a follow-up, if we want to document it reasonably well (e.g. why might one want to try the wasm backend today)

Reply:
I'd prefer to wait until it's less beta - we have a chance to make a good first impression of the wasm backend path.